Mixpanel/Mixpanel.m

```diff
@@ -1575,6 +1567,22 @@ - (void)markNotificationShown:(MPNotification *)notification
     [self trackNotification:notification event:@"$campaign_delivery"];
 }
 
+#pragma mark - Logging
+- (void)setEnableLogging:(BOOL)enableLogging {
+    gLoggingEnabled = enableLogging;
+    
+    if (gLoggingEnabled) {
+        asl_add_log_file(NULL, STDERR_FILENO);
+        asl_set_filter(NULL, ASL_FILTER_MASK_UPTO(ASL_LEVEL_DEBUG));
+    } else {
+        asl_remove_log_file(NULL, STDERR_FILENO);
```

Comment:
This is where most of the magic happens. This redirects `ASL` to the Standard Error Output which will make it viewable to developers in the Xcode console.